.dockerignore

```diff
@@ -5,6 +5,6 @@
 !/benchmarks/
 !/composer.json
 !/phpunit.xml.*
-!/phpstan.neon.*
+!/phpstan*.neon
```

Comment:
why this change?

Reply:
@mcg-web I think because i renamed the `phpstan.neon.dist` long time ago

Reply:
Yes the name changed and the config wasn't included

Reply:
ok nice, thank you!